vars/RedHat.yml

```diff
@@ -28,3 +28,6 @@ __php_opcache_conf_filename: 10-opcache.ini
 __php_fpm_daemon: php-fpm
 __php_fpm_conf_path: "/etc/fpm"
 __php_fpm_pool_conf_path: "/etc/php-fpm.d/www.conf"
+
+__php_fpm_pool_user: apache
```

Comment:
Hmm... it would only be `apache` if you're using Apache as a webserver, right?

I have `www-data` set on my Drupal VM instance running CentOS 7, and it seems to work okay.